content/en/graphing/metrics/distributions.md

```diff
@@ -46,51 +46,28 @@ To customize tagging, hover over your metric in the table, and click on the penc
 ## Case Studies
 Distribution metrics are a new type of metric that allows you to obtain the global distribution values across all hosts. Therefore, metrics that are submitted as distribution metrics create custom metrics the same way as other metric types.
 
-`my.service.latency` is a metric that is being submitted on 500 hosts.
+`my.service.latency` is a distribution metric that is being submitted on 500 hosts.
 
 Each host is tagged with one of 3 `Availability Zones` (as tagged by the AWS integration) and 20 `Roles` by Chef, Datadog provisioning system.
 
-Additionally, this metric has been tagged with `Status`, which has 2 values: `Status:Success` or `Status:Fail`, and `Result`, which also has 2 values: `Result:Open` or `Result:Close`.
+Additionally, this metric has been tagged with `Status`, which has 2 values: `Status:Success` or `Status:Fail`, and `Result`, which also has 2 values: `Result:Open` or `Result:Close`. For the following scenarios, we'll assume that both values of the `Status` tag appear with both values of the `Result` tag.
 
 ##### Scenario 1
+Suppose `my.service.latency` is a distribution metric tagged by `Status` and `Result` that only needs to queried with the nonpercentile aggregations such as avg, min, max, sum, count.
 
-By default, Datadog creates aggregations for `my.service.latency` for each combination of custom metric tags `Status` and `Result`.
+Datadog stores 4 timeseries (one for each nonpercentile aggregation: min, max, sum, count -- avg is calculated as the quotient of sum/count) per unique tag value combination that appears. In this case, since every value of `Status` appears with every value of the `Result` tag. 
```

Comment:
```suggestion
Datadog stores four timeseries (one for each nonpercentile aggregation: `min`, `max`, `sum`, `count` -- `avg` is calculated as the quotient of sum/count) per unique tag value combination that appears. In this case, since every value of `Status` appears with every value of the `Result` tag. 
```